src/components/MultiButtonWithSubQuery.tsx

```diff
@@ -31,8 +45,21 @@ export const navigateWithSubQuery = ({
     });
   }
 
-  // if the `params` is an object, it contains a `routeName` and a `webUrl`
   if (!!params && typeof params === 'object') {
+    // if `params` is an object and contains the `paramsForButton` object,
+    // the values here are added to the navigation
+    if (params.paramsForButton) {
+      return navigation.navigate({
```

Comment:
If we are on a Html-Screen and navigate to a Html-Screen there is no navigation but exchange of screen contents and we cannot navigate back. We will need `push`s here instead of `navigate`s, to ensure clear navigation through the app.

Reply:
6169dd9

https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/50082c0d-e518-49c1-b884-276c4aa975fc

Reply:
Thank you very much for the change. 👍